Aflores ce procedures r2 (#759)
osisoft/OCS-Docs: content/add-organize-data/collect-data/connectors/pi-to-ocs/set-up-pi-to-ocs/config-utility.md

1. To keep the current AF server configuration settings and restart the agent, select **Save**.

1. (Optional) To check that the connection to the AF server is working, select **Test Connection**.

1. (Optional) To delete a server connection, select **Remove Server**, then select **Yes** to confirm. 

   **Note:** After you save the AF server configuration settings and restart and register the agent, you need to select a default Data Archive in PI System Explorer to resolve substitution references for AF element attributes.

## Select the default Data Archive in PI System Explorer

You need to specify the default Data Archive, also referred to as the default data server, for the PI system and PI AF database after setting an AF server. By default, PI AF databases inherit the PI AF Server's local default data server. See [Find the default PI Data Archive server](https://docs.osisoft.com/bundle/pi-server/page/find-the-default-pi-data-archive-server.html) for more information.

To select the default Data Archive, follow these steps:

1. Open PI System Explorer on the client machine.

1. Select **File** > **Server Properties**.

> 1. In the `PI AF Server Properties` window, select the **Default Data Server down arrow**, and then select the data server to be the default.<!--AF 11/12/21 what is Default Data Server down arrow???-->

Reviewer: Should this be **Default Data Server* down arrow?

```suggestion
1. In the `PI AF Server Properties` window, select the **Default Data Server** down arrow, and then select the data server to be the default.<!--AF 11/12/21 what is Default Data Server down arrow???-->
```

**Note:** If you are not adding an AF server, select the PI Data Archive icon on the first screen of the PI to OCS Agent Configuration Utility.

To add a PI Data Archive, follow these steps:

1. Open the PI to OCS Agent Configuration Utility.
   
1. Select **Data Archive Server**, then enter the name of the Data Archive server, and then select **Add Server**.
   
   The PI Data Archive connection is added and details about the newly added PI Data Archive are displayed.

1. Review the following details for the PI Data Archive:
 
   * Server name, version, and server ID

   * IP address

   * Connection status and timeout
   
1. (Optional) To change the length of time the agent checks for a server connection before timing out, select the pencil icon next to the **Connection Timeout (sec)** field.

1. (Optional) To confirm that the connection to the Data Archive is working, select **Test Connection**.

1. (Optional) To remove the configured PI Data Archive, select **Remove Server**.

1. To retain the current PI Data Archive configuration, select **Save**.

## Create a PI mapping 

> PI mappings enable access to data stored on a PI Data Archive by service accounts assigned to a PI identity. PI mappings can be created for a PI identity, user, or group. Accounts assigned to a PI identity can read and transfer PI point data to OCS. For more information, see ["What are PI identities and mappings?"](https://livelibrary.osisoft.com/LiveLibrary/content/en/server-v14/GUID-AD57C990-0DE6-4280-B9F3-1432C6E444C5). You can also edit mappings with the utility.

Reviewer: ```suggestion
PI mappings enable access to data stored on a PI Data Archive by service accounts assigned to a PI identity. PI mappings can be created for a PI identity, user, or group. Accounts assigned to a PI identity can read and transfer PI point data to OCS. For more information, see ["What are PI identities and mappings?"](https://docs.osisoft.com/bundle/pi-server/page/what-are-pi-identities-and-mappings_new.html). You can also edit mappings with the utility.
```

![Agent description and hostname displayed in PI to OCS Agents window](../../images/pi-to-ocs-agents-hostname.png)
<!--AF 11/12/21 I would change the callouts in the image to be minimal, like "Agent description" and "host name". Then add any additional explanation here in text. It will be easier to read and easier to maintain. -->

To configure data privacy and add an agent description, follow these steps:

1. Open the PI to OCS Agent Configuration Utility.

1. To open the `PI to OCS Agent Settings` window, select the pencil icon to the right of **Agent Service Account**.

1. To publish the hostname, select the **Opt-in to publishing PI to OCS Agent Hostname in OCS?** option under `Data Privacy`.

1. (Optional) In the **PI to OCS Agent Description** field, enter a descriptive name for the agent.

1. To save your selections, select **Ok** , and then select **Save** in the utility.

## List of agent status states

> It may take a few minutes for a PI System to register with OCS. The table below lists the various states that may appear under the Agent Status field in the PI to OCS Configuration Utility.

Reviewer: ```suggestion
It may take a few minutes for a PI System to register with OCS. The table below lists the various states that may appear under the **Agent Status** field in the PI to OCS Configuration Utility.
```

| **State**                     | **Description**                                              |
| ----------------------------- | ------------------------------------------------------------ |

> | Data  Source Connection Issue | Indicates the PI To OCS Agent cannot connect to the PI Data Archive. Some reasons for this status include the PI Data Archive is turned off, a firewall issue is preventing connections, or an incorrect name is configured for the Data Archive. For example, the agent is trying to connect to a machine that does not exist or was renamed. There may be additional reasons for this status. |

Reviewer: ```suggestion
| Data Source Connection Issue | Indicates the PI To OCS Agent cannot connect to the PI Data Archive. Some reasons for this status include the PI Data Archive is turned off, a firewall issue is preventing connections, or an incorrect name is configured for the Data Archive. For example, the agent is trying to connect to a machine that does not exist or was renamed. There may be additional reasons for this status. |
```

> | Data  Source Security Issue   | Indicates the PI Data Archive connection is unsecured and security settings need to be  addressed. |

Reviewer: ```suggestion
| Data Source Security Issue   | Indicates the PI Data Archive connection is unsecured and security settings need to be addressed. |
```
